react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -668,18 +877,21 @@ function _mapStateToProps(state) {
     const toolbarButtons = getToolbarButtons(state);
     const { testing = {}, iAmRecorder } = state['features/base/config'];
     const enableThumbnailReordering = testing.enableThumbnailReordering ?? true;
-    const { visible, remoteParticipants } = state['features/filmstrip'];
+    const { visible, remoteParticipants, width: verticalFilmstripWidth } = state['features/filmstrip'];
     const reduceHeight = state['features/toolbox'].visible && toolbarButtons.length;
     const remoteVideosVisible = shouldRemoteVideosBeVisible(state);
     const { isOpen: shiftRight } = state['features/chat'];
     const {
-        gridDimensions = {},
+        gridDimensions: dimensions = {},
```

Comment:
why do we need alias here? below you again assign `let gridDimensions = dimensions`. I think you should do a spread of the gridDimensions on another var in case you don't want to alter the original object instead of aliasing, as aliasing does keep the reference.

Reply:
It's because gridDimensions can be reassigned, so it has to be let, not const